opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/SpanWrapper.java

```diff
@@ -44,6 +48,7 @@ protected void finish() {
         boolean wasFinished = finished.getAndSet(true);
         if (!wasFinished) {
```

Comment:
not related to this but could you please rewrite this to:

`if (finished.compareAndSet(false,true))`